src/index.html

```diff
@@ -32,6 +32,1139 @@
   <link rel="apple-touch-icon-precomposed" sizes="120x120" href="./img/components/icons/favicon-apple-120x120.png">
   <link rel="apple-touch-icon-precomposed" sizes="152x152" href="./img/components/icons/favicon-apple-152x152.png">
   <script type="text/javascript">(function () { var PAGE_LOADED_TIME = Date.now(); window.getPageLoadedTime = function () { return PAGE_LOADED_TIME; } })();</script>
+  <script>
```

Comment:
Why was this added here and not imported somewhere else?

Reply:
There's an issue with `reflect-metadata` and `webpack@3` if you do it the right way it doesn't work. This is a workaround I've found. They say with `webpack@4` we could use `DefinePlugin`

Reply:
Awesome, could you add a Jira issue for that, so we don't forget?